dubbo-admin/src/main/resources/i18n/message.properties

```diff
@@ -14,4 +14,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
+dependencies=
```

Comment:
what's the purpose for this file change? my understanding is we need have different i18n information in message_{lang}.properties file

Reply:
Yes.It is.
message.properties‘s content is auto generated by IDEA Resource Bundle.I didn't have a attention. It had been removed.